csuntechlab/iSTART
# STR-128 --FE: Create informational slide template

# Description
'Write a meaningful description of your task'
  Created a template that reflects the informational slides.
  Should include styling as well as the functionality
  Should be mobile responsive
  Our documents mention a carousel, do not worry about carousel.

# Testing
'What does the reviewer need to do to test your PR?'
 - Login as `april.feldman` 
 - Click on a module
 - You need to make sure that the info template looks like the mockup
 - Be as constructively critical as possible on my code and on my styling
 - Make sure that the user is only able to proceed after five seconds
